dev-client/src/screens/LoginScreen.tsx

```diff
@@ -105,34 +147,13 @@ export const LoginScreen = () => {
           </Heading>
         </Column>
         <Column alignItems="center" flexGrow={1}>
-          <Button.Group direction="column" space={5}>
-            <Button
-              style={styles.loginButton}
-              _text={styles.loginButtonText}
-              size="lg"
-              onPress={onPress('google')}
-              startIcon={<GoogleLogo />}>
-              {t('account.google_login')}
-            </Button>
-            <Button
-              style={styles.loginButton}
-              _text={styles.loginButtonText}
-              size="lg"
-              onPress={onPress('microsoft')}
-              startIcon={<MicrosoftLogo />}>
-              {t('account.microsoft_login')}
-            </Button>
-            {showAppleAuth ? (
-              <AppleAuthenticationButton
-                buttonType={AppleAuthenticationButtonType.SIGN_IN}
-                buttonStyle={AppleAuthenticationButtonStyle.WHITE}
-                style={styles.appleloginButton}
-                onPress={onPress('apple')}
-              />
-            ) : (
-              <></>
-            )}
-          </Button.Group>
+          {!isLoading && <LoginButtons onPress={onPress} />}
+          {isLoading && (
+            <ActivityIndicator
+              size="large"
+              color={theme.colors.primary.contrast}
+            />
+          )}
```

Comment:
suggestion: use a ternary instead of two separate checks.
```suggestion
          {isLoading ? (
            <ActivityIndicator
              size="large"
              color={theme.colors.primary.contrast}
            />
          ) : 
          <LoginButtons onPress={onPress} />
        }
```